centaur/src/main/resources/standardTestCases/cwl_dynamic_initial_workdir.test

```diff
@@ -0,0 +1,17 @@
+name: cwl_dynamic_initial_workdir
+testFormat: workflowsuccess
+workflowRoot: main
+workflowType: CWL
+workflowTypeVersion: v1.0
+backendsMode: "only"
+backends: [Local, LocalNoDocker]
+tags: [localdockertest]
+
+files {
+  workflow: cwl_dynamic_initial_workdir/cwl_dynamic_initial_workdir.cwl
+  inputs: cwl_dynamic_initial_workdir/cwl_dynamic_initial_workdir.inputs
+}
+
+metadata {
+  status: Succeeded
```

Comment:
Is there anything else this test can/should validate?

Reply:
If nothing exploded that's a big upgrade from what would happen on current develop, but let me see if I can do even better.